daprdocs/content/en/reference/arguments-annotations-overview.md

```diff
@@ -32,28 +32,28 @@ This table is meant to help users understand the equivalent options for running
 | `--log-as-json` | not supported | | `dapr.io/log-as-json` | Setting this parameter to `true` outputs logs in JSON format. Default is `false` |
 | `--log-level` | `--log-level` | | `dapr.io/log-level` | Sets the log level for the Dapr sidecar. Allowed values are `debug`, `info`, `warn`, `error`. Default is `info` |
 | `--enable-api-logging` | `--enable-api-logging` | | `dapr.io/enable-api-logging` | Enables API logging for the Dapr sidecar |
-| `--app-max-concurrency` | `--app-max-concurrency` | | `dapr.io/app-max-concurrency` | Limit the concurrency of your application. A valid value is any number larger than `0`
+| `--app-max-concurrency` | `--app-max-concurrency` | | `dapr.io/app-max-concurrency` | Limit the concurrency of your application. A valid value is any number larger than `0`|
 | `--metrics-port` | `--metrics-port` | | `dapr.io/metrics-port` | Sets the port for the sidecar metrics server. Default is `9090` |
 | `--mode` | not supported | | not supported | Runtime mode for Dapr (default "standalone") |
-| `--placement-address` | `--placement-address` | | not supported | Addresses for Dapr Actor Placement servers |
+| `--placement-address` | `--placement-address` | | dapr.io/placement-addresses | Addresses for Dapr Actor Placement servers. When no annotation is set, the default value is set by the Sidecar Injector. When the annotation is set and the value is empty, the sidecar will skip connecting to placement. When the annotation is set and the value is non empty, the sidecar will connect to the configured address. |
```

Comment:
```suggestion
| `--placement-address` | `--placement-address` | | dapr.io/placement-addresses | Addresses for Dapr actor Placement servers. When no annotation is set, the default value is set by the Sidecar Injector. When the annotation is set and the value is empty, the sidecar does not connect to Placement server. This can be used when there are no actors running in the sidecar. When the annotation is set and the value is not empty, the sidecar connects to the configured address. |
```

Reply:
Some suggest edits above. 
Question: Can this be a list of placement addresses? If so that we should say a list
Also can you given an example of the address, for example does this need a IP:port combination?

Reply:
@msfussell Done. Please re-review.

Reply:
One more question: Does the list of Placement addresses also work for the CLI? Can I have a comma separated list in the --placement-address CLI flag?

Reply:
> Yes, it is also supported. I found that `--placement-address` has been replaced by `--placement-host-address` @msfussell

Well that is strange. I updated this PR to include "host" in the name, however we should now change the K8s annotation to be the same name, given that we have just introduced this. @CrazyHZM Can you raise an issue in the runtime repo to change the name and when fixed we can then update this PR with the change?

Reply:
Of course, I will discuss the issue under this PR：https://github.com/dapr/dapr/pull/4094